target/linux/mediatek/image/mt7623.mk

```diff
@@ -33,21 +45,53 @@ define Device/bpi_bananapi-r2
   DEVICE_PACKAGES := kmod-fs-vfat kmod-nls-cp437 kmod-nls-iso8859-1 kmod-mmc \
 	mkf2fs e2fsprogs kmod-usb-ohci kmod-usb2 kmod-usb3 kmod-ata-ahci-mtk
   UBOOT_TARGET := mt7623n_bpir2
+  UBOOT_OFFSET := 320k
+  UBOOT_ENVSIZE := 0x2000
```

Comment:
Are these added to DEVICE_VARS somewhere? (Haven't checked myself)

Reply:
They are now.